### DISCUSSION TITLE
docs/UCD/Ant/README

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#19] [Q&A]
# docs/UCD/Ant/README

Apache Ant is a command line tool that is used to automate the build and deployment of applications. This plug-in calls the Ant executable to run one or more targets defined in a specified Ant script ...

http://127.0.0.1:5500/docs/UCD/Ant/README.html

**Comment by jmanojhcl** [comment]
Test comment
